app/src/main/res/values/strings.xml

```diff
@@ -409,6 +409,7 @@
   <string name="scansimpleid_enteredcode_error_empty">Enter BP Passport number or a National Health ID</string>
   <string name="scansimpleid_enteredcode_error_not_required_length">Enter a valid BP Passport number or a National Health ID</string>
   <string name="scansimpleid_type_number">or type number</string>
+  <string name="scansimpleid_invalid_qr_code">Scan a valid health ID</string>
```

Comment:
Shouldn't the error be `Scan a valid National Health ID or BP passport` ? Because we're showing this error for all countries hence `scan a valid health ID` doesn't make sense. WDYT?

Reply:
We had a conversation regarding this [here](https://simpledotorg.slack.com/archives/CFM321423/p1621328194002200)